src/containers/upcoming-events/ducks/types.ts

```diff
@@ -28,4 +29,5 @@ export interface EventProps {
     end: Date;
   };
   price: number;
+  announcements: EventAnnouncement[];
```

Comment:
Wait, this doesn't seem right at all. We aren't storing the event announcements on each event. This says that each Event has a list of announcements, which is not true. 

I think the solution here is that the component needs its own interface, which can extend this interface we'll use in the store for UpcomingEvents.